Frontend/ui-driver/src/Flow.purs

```diff
@@ -1198,7 +1201,20 @@ homeScreenFlow = do
       _ <- pure $ setValueToLocalStore DRIVER_STATUS_N "Online"
       _ <- pure $ setValueToLocalNativeStore DRIVER_STATUS_N "Online"
       (DriverActiveInactiveResp resp) <- Remote.driverActiveInactiveBT "true" $ toUpper $ show Online
-      _ <- pure $ firebaseLogEvent "ny_user_ride_completed"
+      _ <- pure $ firebaseLogEvent "ny_user_ride_completed" 
+
+      if (getValueToLocalStore HAS_TAKEN_FIRST_RIDE == "true") then do
+        getDriverInfoResp <- Remote.getDriverInfoBT (GetDriverInfoReq { })
+        let (GetDriverInfoResp getDriverInfoResp) = getDriverInfoResp
+        if (isJust getDriverInfoResp.numberOfRides) then do
```

Comment:
can you add a case match instead of so many if else statements?

Reply:
Sure, done